src/CoreBundle/private/library/classes/TViewParser/TViewPathManager.class.php

```diff
@@ -102,17 +102,20 @@ public function getWebModuleViewPath($sViewName, $sModuleName, $sType = 'Custome
         /*
          * @deprecated only used by pkgBlog
          */
+        $sPath = TPkgViewRendererSnippetDirectory::PATH_MODULES;
```

Comment:
The comment states that this code fragment is deprecated. Did you change it just for completeness' sake (which is OK) or is the code actually used (so that we should think about the deprecation message)?

Reply:
The code is actually use if the template was not found.